### ISSUE TITLE
Expose content rating

### ISSUE THREAD

**Issue by elementaryBot** [elementary/appcenter#34]
Currently GNOME Software is using https://odrs.gnome.org/oars
It seems to provide a wide range of granularity but more is better than less here

It would be nice to expose it to the user.

~A nice to have would be to be able to restrict certain categories in the Parental Control Plug~ See #629 

Launchpad Details: [#LP1665076](https://bugs.launchpad.net/bugs/1665076) Corentin Noël - 2017-02-15 18:55:59 +0000

**Comment by cassidyjames** [comment]
I think this needs to be broken up a bit. 

- [x] Encourage devs to use OARs data
- [ ] Displaying the content rating in AppCenter
- [ ] Provide an API for restricting based on the rating

I think this issue is the second item, and #629 would be the third.

**Comment by danrabbit** [comment]
Just fleshing this out a bit, these are the categories of content that OARS currently deals with. It goes into a lot more detail and granularity, but I think some way to represent that the app contains "Some form of the following" would be a helpful first step. I've made some notes about what is included in each category and some thoughts about maybe ignoring certain things so that we give useful/accurate content warnings and don't go overboard. I think warning fatigue is something we should watch out for:

* **Violence**. This covers cartoon and fantasy violence as well as realistic violence and violence-related concepts like slavery and desecrating religious symbols, buildings, etc.
* **Drugs**. This includes both the presence of or references to alcohol, narcotics, or tobacco. This might be better defined as "illicit substances" or some such other since depending on your locale you might not consider all of these to be "drugs" in the traditional sense of the word. 
* **Sex & Nudity**. This oddly includes references to homosexuality and the vague "sexual appearance" with the provided example being women's swimwear. This one might need some special casing to make sure it's useful as a broad category. I wouldn't expect a game like Pokémon to contain a content warning for "Sex & Nudity" because some characters are in swimwear.
* **Language**. This includes the obvious profanity, but also discrimination and humor. Again might need some filtering since I don't think a "Language" content warning makes sense if your app contains "slapstick humor".
* **Money**. This includes advertising, gambling, and in-app purchases. In this context advertising goes so far as to refer to "banners showing the Coca-Cola logo shown in a Soccer game". Since we don't allow what most people would consider advertising in AppCenter, maybe we should just look for gambling and in-app purchases here. iOS places a warning about in-app purchases right next to the price button, which I think is an interesting choice especially when the app shows as free. It's kind of a nice way to say "Free, but here's the catch". If we did something like that, then we could just warn about gambling on its own.
* **Social**. This basically covers anything where you send data to the internet for other humans to see.

My guess is we could show something like the PEGI rating icons on the page somewhere. Here are examples of those:

![](http://www.conducttr.com/assets/2015-06-10_12-18-36-755x294.jpg)
![](http://www.swedishgamesindustry.com/umbraco/ImageGen.ashx?image=%2Fmedia%2Fnews_pic%2Fthumbmaker1.png&width=640&height=300&constrain=true)

It might actually be worth proposing a standard set of icons upstream for OARS.

Interestingly, GNOME Software seems to be [generating an actual age rating](https://blogs.gnome.org/hughsie/2016/09/21/gnome-software-and-age-ratings/) though I don't see anything that looks like a spec for this on their website. I'm not sure how terribly useful this is since what kind of content parents are okay with seems to be a pretty personal choice.
